data_server/Dockerfile

```diff
@@ -9,9 +9,13 @@ ENV PYTHONUNBUFFERED True
 ENV APP_HOME /app
 WORKDIR $APP_HOME
 COPY ./data_server ./data_server
+COPY ./python/data_server ./python/data_server
 
 # Install production dependencies.
+RUN python -m pip install --upgrade pip
```

Comment:
I added it because when I was building my containers it was giving a warning that it wasn't using the latest version of pip. I think it would make sense to add it to the other Dockerfiles. I'll send that as a followup CL.